src/screens/Home/styles.ts

```diff
@@ -0,0 +1,27 @@
+import { Colors } from "@constants/constants";
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: "center",
+    alignItems: "center",
+    padding: 16,
+  },
+  loadingContainer: {
+    flex: 1,
```

Comment:
flex: 1,
    justifyContent: "center",
    alignItems: "center",
    
    create a global view style for this

Reply:
okkkkkk
working on it